android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -393,7 +394,7 @@ public void onBandwidthSample(int elapsedMs, long bytes, long bitrate) {
                 int width = videoFormat != null ? videoFormat.width : 0;
                 int height = videoFormat != null ? videoFormat.height : 0;
                 String trackId = videoFormat != null ? videoFormat.id : "-1";
-                eventEmitter.bandwidthReport(bitrate, height, width, trackId);
+                eventEmitter.bandwidthReport(bitrate, height, width, String.valueOf(hdcp));
```

Comment:
we're just in the process of trying to unpick all the PRs to this repo, I don't think we should add more.

Reply:
sure I will close this pr only test on my feature branch for now